src/screens/Registration/index.jsx

```diff
@@ -58,14 +61,11 @@ const Registration = () => {
     try {
       event.preventDefault();
       setIsLoading(true);
-      const response = await httpPost('user', registrationData);
-      if (!response.errors) await login();
+      const response = await httpPost('auth/signup', registrationData);
+      if (!response.error) await login();
```

Comment:
lets do signup without login
means that signup should return token like in login logic
no needed have 2 request here